docs/resources/pre_authorized_application.md

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Applications"
+---
+
+# Resource: azuread_pre_authorized_application
+
+Manages client applications that are pre-authorized with the specified permissions to access an application's APIs without requiring user consent.
+
+## Example Usage
+
+```terraform
+resource "azuread_application" "authorized" {
+  display_name = "example-authorized-app"
+}
+
+resource "azuread_application" "authorizer" {
+  display_name = "example-authorizing-app"
+
+  api {
+    oauth2_permission_scope {
+      admin_consent_description  = "Administer the application"
+      admin_consent_display_name = "Administer"
+      enabled                    = true
+      id                         = "ced9c4c3-c273-4f0f-ac71-a20377b90f9c"
+      type                       = "Admin"
+      value                      = "administer"
+    }
+
+    oauth2_permission_scope {
+      admin_consent_description  = "Access the application"
+      admin_consent_display_name = "Access"
+      enabled                    = true
+      id                         = "2d5e07ca-664d-4d9b-ad61-ec07fd215213"
+      type                       = "User"
+      user_consent_description   = "Access the application"
+      user_consent_display_name  = "Access"
+      value                      = "user_impersonation"
+    }
+  }
+}
+
+resource "azuread_pre_authorized_application" "example" {
```

Comment:
should this be renamed to
```suggestion
resource "azuread_application_pre_authorized" "example" {
```

to sort better with application?

Reply:
Good call, will do 👍